### PR TITLE
Just fix bb, already.

### PR DESCRIPTION
**Pull request by rminnich**
This merges two needed fixes and the first of the two didn't even work.

### CODE REVIEW

File: bb/ramfs.go [hunk 1]
```diff
@@ -25,7 +24,7 @@ type copyfiles struct {
 }
 
 const (
-	bbList = `{{.Uroot}}/bb/bbsh
+	bbList = `{{.Gopath}}/github.comd/src/u-root/u-root/bb/bbsh
```
Comment (hugelgupf) [inline bb/ramfs.go]: .com, not .comd

Comment (rminnich) [inline bb/ramfs.go]: done.

**Comment by rminnich** [comment]
bump

**Comment by hugelgupf** [comment]
filepath should not be necessary unless you are trying to build/use u-root on something that doesn't use forward-slash as the path separator. Haven't tried it, but LGTM for now if it works for you.

**Comment by rminnich** [comment]
I moved to filepath for everything when iant dinged me for not using it on an upstream CL for go. 
"if it's an operation on file paths, use filepath". Might as well use it if it's the right thing to use. THanks for the review.

**Pull request merged**